packages/salesforcedx-vscode-core/test/vscode-integration/commands/baseDeployRetrieve.test.ts

```diff
@@ -569,62 +569,6 @@ describe('Base Deploy Retrieve Commands', () => {
         expect(appendLineStub.calledOnce).to.equal(true);
         expect(appendLineStub.firstCall.args[0]).to.equal(expectedOutput);
       });
-
```

Comment:
Since we are not fixing the mocha test or replacing it with jest test, let's not remove the test. Eith 'x' it or comment out the test. 
`xit('should......')` will prevent it from running.